modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,38 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta processor
+
+The Cumulative to Delta processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
```

Comment:
I read the notice about the tech preview admonition. Just to clarify that the tech preview snippet will be used in the other PR?

For example:

:FeatureName: Using Krew to install and manage plugins for the OpenShift CLI
include::snippets/technology-preview.adoc[]

Reply:
Does the TP fix PR exist? I cannot find it? Wondering if the issue is build or merge conflict related?

Reply:
@dfitzmau, I will not be merging this PR directly. I will cherry-pick this PR's commit into one XXXL PR of the next release of this product, and then I will close this PR. I haven't opened the TP admonition fix PR yet, but it will also be merged into the XXXL PR of the next release. Please note that the TP admonition is currently not used throughout the published page. I'm asking for peer reviews on smaller PRs such as this to simplify reviews for peer reviewers (as opposed to having to review an XXXL PR) and to keep the release date manageable.

Reply:
I've amended this PR.

Reply:
Thanks. Yeah, this is a good approach to manage the work as XL+ PRs are not very well liked. Thanks for adding the TP note. This helps my peer review as a [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features) is now met.

No direct guidance exists for starting a module with an admonition but the RH SSG does mention the [guidance for lead-in](https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences) sentences for a module. Add a lead-in sentence and then add the TP snippet after it? Search engine optimization (SEO) results usually truncate the first sentence or two of a module (in our case) so the "Important" part would be incorporated into that short sentence segment.

Reply:
Lead-in consideration (although adding more of a "why" a customer would use it, might be a useful addition):

> The Cumulative to Delta processor converts monotonic, cumulative-sum, and histogram metrics to monotonic delta metrics.

[TP SNIPPET]

Reply:
> Thanks. Yeah, this is a good approach to manage the work as XL+ PRs are not very well liked. Thanks for adding the TP note. This helps my peer review as a [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features) is now met.
> 
> No direct guidance exists for starting a module with an admonition but the RH SSG does mention the [guidance for lead-in](https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences) sentences for a module. Add a lead-in sentence and then add the TP snippet after it? Search engine optimization (SEO) results usually truncate the first sentence or two of a module (in our case) so the "Important" part would be incorporated into that short sentence segment.

Thank you for the tip. I'll see how I can incorporate it into a global revision that I plan for this file after the upcoming release (this file is very long, so there are more lines to correct that are beyond the scope of this PR).